src/launch/launch_config.cpp

```diff
@@ -563,6 +574,13 @@ void LaunchConfig::parseROSParam(TiXmlElement* element, ParseContext ctx)
 			buffer << stream.rdbuf();
 
 			contents = buffer.str();
+
+			if(ctx.checkEmptyString(contents))
+			{
+				printf("File %s is empty, skipping...\n", fullFile.c_str());
```

Comment:
If roslaunch does not print a warning here, we should not either. So just silently ignore the file.